packages/contracts/deploy/30_plugins/10_plugin-repos/00_create-plugins-repo.ts

```diff
@@ -13,18 +13,16 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   // AddresslistVotingSetup
   await createPluginRepo(
     hre,
-    'AddresslistVoting',
-    'AddresslistVotingSetup',
-    [1, 0, 0],
+    'AllowlistVoting',
+    'AllowlistVotingSetup',
     '0x'
   );
 
   // TokenVotingSetup
   await createPluginRepo(
     hre,
-    'TokenVoting',
-    'TokenVotingSetup',
-    [1, 0, 0],
+    'ERC20Voting',
+    'ERC20VotingSetup',
```

Comment:
Why this name? The plugin is now named TokenVoting

Reply:
This should be done in separate PR. nothing related to this PSP. I merged `develop` into this so if it still like this, it's develop branch problem(completely different task so we don't do everything in this PR).

Reply:
Oh, my bad. Overlooked. Good catch. Fixed https://github.com/aragon/core/pull/164/commits/b647ff5683721f3c60f8e75710d675d1141d9943